bin/build-image

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+###############################################################################
+#
+# Build a Docker image of the SMAP-L2-Gridder service.
+#
+###############################################################################
+
+image="ghcr.io/nasa/smap-l2-gridder"
+
+# Retrieve the tag from the script arguments, or default to "latest". Bamboo
+# will pass the contents of `docker/service_version.txt` as this argument,
+# which contains a semantic version number.
+#
+tag=${1:-latest}
+
+
+# Remove old versions of: ghcr.io/nasa/smap-l2-gridder and
+# ghcr.io/nasa/smap-l2-gridder-test images
+./bin/clean-images
+
+# Build the image
+# This command tags the image with two tags:
+# - The one supplied as a script argument. Bamboo will provide the semantic
+#   version number from `docker/service_version.txt`.
+# - "latest", so the test Dockerfile can use the service image as a base image.
+#
+docker build  -t ${image}:${tag} -t ${image}:latest -f docker/service.Dockerfile .
```

Comment:
I am purposefully not including the `--platform` flag because this code compiles natively on both ARM and AMD machines.

Reply:
Not exactly, but because there are no python dependencies that can't be built on the ARM natively.